static/css/explore.scss

```diff
@@ -251,23 +251,23 @@
 }
 
 .loading-container {
```

Comment:
just want to make sure -- did you check to see what/if other components use this class and if it has any unintended consequences?